lib/include/nitro/fx/fx_mtx44.h

```diff
@@ -0,0 +1,82 @@
+#ifndef NITRO_FX_FX_MTX44_H_
+#define NITRO_FX_FX_MTX44_H_
+
+#include <nitro/mi/memory.h>
+#include <nitro/fx/fx.h>
+#include <nitro/fx/fx_const.h>
+
+static inline void MTX_Identity44(MtxFx44 * pDst);
```

Comment:
MTX is the erong prefix for these funcs, it should be FX

Reply:
pointer style does not follow the established convention

Reply:
also inlines with no content

Reply:
look I'm not ok with this entire file, and I presume the c file associated with it
I'm presuming this is copied and pasted, I'm already a bit unsure about single func copy and pastes, but an entire file? this isn't right, it needs to be *at least* edited to confirm with the established style and conventions
what's worse is this file already *exists* in diamond and is decomped properly
if you have to copy smth at least copy from there